qcodes/instrument_drivers/Keysight/Keysight_33500B.py

```diff
@@ -36,21 +36,6 @@ def setcmd(channel, setting):
         def getcmd(channel, setting):
             return 'SOURce{}:'.format(channel) + setting + '?'
 
-        def errorparser(rawmssg):
```

Comment:
Is it not the time to just remove this driver?

Reply:
ok, i'll remove it.

Reply:
I won't remove it because it requires updating some example notebooks, i'd prefer to do that in a different PR.